.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+services:
+  - docker
+python:
+  - 3.6
+env:
+  - KITCHEN_YAML=.kitchen.docker.yml
+script:
+  # Install dependencies
+  - gem install bundler
+  - bundle update
+  # Setup for tests
+  - bundle exec kitchen create -l warn -c 4
+  # Provision containers
+  - bundle exec kitchen converge
+  # Run test-kitchen with docker driver:
+  - bundle exec kitchen verify
+after_script:
+  - bundle exec kitchen list
+  - bundle exec kitchen destroy
```

Comment:
Hi @daks, perhaps you can use the files in the [packages formula](https://github.com/saltstack-formulas/packages-formula/) or the [apache formula](https://github.com/saltstack-formulas/apache-formula/) as a reference?

Ie, the [.travis.yml file](https://github.com/saltstack-formulas/apache-formula/blob/master/.travis.yml) can be greatly simplified and little is needed to get it up and running.